src/components/MnemonicPhrase/styled.tsx

```diff
@@ -3,47 +3,29 @@ import { em } from 'polished';
 
 import { boxSizing } from '../../modules/box-sizing';
 
-export const Container = styled.ol`
+export const Container = styled.div`
   ${boxSizing};
 
   counter-reset: listCounter;
-  display: flex;
-  list-style: none;
-  margin: -${({ theme }) => theme.honeycomb.size.reduced};
-  padding: 0;
-  align-items: center;
-  justify-content: center;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(${em(100)}, 1fr));
 `;
 
-export const Item = styled.li`
+export const Item = styled.div`
   counter-increment: listCounter;
-  display: flex;
-  min-width: 50px;
-  background: ${({ theme }) => theme.honeycomb.color.bg.masked};
-  color: ${({ theme }) => theme.honeycomb.color.readable.normal(theme.honeycomb.color.bg.masked)};
-  border: 1px solid ${({ theme }) => theme.honeycomb.color.border};
-  border-radius: ${({ theme }) => em(theme.honeycomb.size.huge / 2)};
-  flex-grow: 1;
-  flex-shrink: 0;
-  align-items: center;
-  justify-content: flex-start;
-  margin: ${({ theme }) => em(theme.honeycomb.size.reduced)};
-  padding-right: ${({ theme }) => em(theme.honeycomb.size.increased)};
-  height: ${({ theme }) => em(theme.honeycomb.size.huge)};
+  white-space: nowrap;
   overflow: hidden;
+  text-overflow: ellipsis;
+  color: ${({ theme }) => theme.honeycomb.color.text.normal};
+  margin: ${({ theme }) => em(theme.honeycomb.size.reduced)};
+  font-size: ${({ theme }) => em(theme.honeycomb.size.small)};
+  font-weight: 600;
 
   ::before {
-    content: '#' counter(listCounter);
-    font-size: ${({ theme }) => em(theme.honeycomb.size.reduced)};
-    min-width: ${({ theme }) => em(theme.honeycomb.size.huge, theme.honeycomb.size.reduced)};
-    height: ${({ theme }) => em(theme.honeycomb.size.huge, theme.honeycomb.size.reduced)};
-    background: ${({ theme }) => theme.honeycomb.color.bg.normal};
-    display: flex;
-    padding: 0 ${({ theme }) => em(theme.honeycomb.size.reduced, theme.honeycomb.size.reduced)};
-    margin-right: ${({ theme }) =>
-      em(theme.honeycomb.size.increased, theme.honeycomb.size.reduced)};
-    align-items: center;
-    justify-content: center;
+    content: '#' counter(listCounter, decimal-leading-zero);
+    min-width: ${({ theme }) => em(theme.honeycomb.size.normal, theme.honeycomb.size.reduced)};
+    margin-right: ${({ theme }) => em(theme.honeycomb.size.reduced)};
+    color: ${({ theme }) => theme.honeycomb.color.text.masked};
+    font-size: ${({ theme }) => em(theme.honeycomb.size.small)};
```

Comment:
This is affected by the `font-size` on line 21, so the size will be `9px`.

I checked the design and the size is supposed to be `10px`, which does not exist in the theme. Personally, I would do:

```suggestion
    font-size: ${({ theme }) => em(theme.honeycomb.size.tiny, theme.honeycomb.size.small)};
```

But we could make an exception and do:

```suggestion
    font-size: ${({ theme }) => em(10, theme.honeycomb.size.small)};
```

I'll leave it up to you.

Reply:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/15721063/88375414-a4900080-cdef-11ea-8769-e39623baf227.png">
<img width="377" alt="image" src="https://user-images.githubusercontent.com/15721063/88375723-34ce4580-cdf0-11ea-863f-c8c73b21d68b.png">

I used the `10` option just because `theme.honeycomb.size.tiny` is a bit hard to read.
Also added a fix to vertically align the numbers to the top of the line, to match the design. This is more obvious now that the numbers are smaller.